Sources/Build/Triple.swift

```diff
@@ -129,6 +133,7 @@ public struct Triple: Encodable {
     public static let arm64Linux = try! Triple("aarch64-unknown-linux")
     public static let armLinux = try! Triple("armv7-unknown-linux-gnueabihf")
     public static let android = try! Triple("armv7-unknown-linux-androideabi")
```

Comment:
This was added years back and never used anywhere, I can change it to `armAndroid` to match the other variables' naming if wanted.

Reply:
Sure